desafio-01/README.en.md

```diff
@@ -0,0 +1,80 @@
+<img alt="GoStack" src="https://storage.googleapis.com/golden-wind/bootcamp-gostack/header-desafios.png" />
+
+<h3 align="center">
+  Challenge 1: Planning my studies
+</h3>
+
+<p align="center">“Your only limitation is yourself”!</blockquote>
+
+<p align="center">
+  <img alt="GitHub language count" src="https://img.shields.io/github/languages/count/rocketseat/bootcamp-gostack-desafios?color=%2304D361">
+
+  <a href="https://rocketseat.com.br">
+    <img alt="Made by Rocketseat" src="https://img.shields.io/badge/made%20by-Rocketseat-%2304D361">
+  </a>
+
+  <img alt="License" src="https://img.shields.io/badge/license-MIT-%2304D361">
+
+  <a href="https://github.com/Rocketseat/bootcamp-gostack-desafios/stargazers">
+    <img alt="Stargazers" src="https://img.shields.io/github/stars/rocketseat/bootcamp-gostack-desafios?style=social">
+  </a>
+</p>
+
+<p align="center">
+  <a href="#rocket-about-the-challenge">About the challenge</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
+  <a href="#calendar-deliver">Deliver</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
+  <a href="#memo-licence">Licence</a>
+</p>
+
+## :rocket: About the challenge
+
+In this challenge, you must plan your studies.
+
+For that, you must create a cronogram that will be adapted according to your time and rhythm.
+
+Define what'll be your study schedule, creating tasks that will be attached to a calendar, to help you keep focus and always aligned with your objectives!
+
+### Cronogram template
+
+To help you with that challenge, we created a template that you can duplicate through the Notion platform.
+Notion works either on mobile platforms or desktop platforms, and both can be downloaded in the [Oficial Page](https://www.notion.so/product).
+
+To start using the template, you must access the link above and click in the button "Duplicate" which is in the menu at the top right of the screen.
+
+📄 **[Link to the template](https://www.notion.so/Cronograma-de-estudos-e390bc8d2f5743668ec03348a3306070)**
+
+### Defining a weekly schedule
+
+<p align="center">
+  <img width="80%" src="./assets/cronograma-diario.png">
+</p>
+
+In this section you must inform a schedule that you will dedicate during your journey as a developer, after all, the studies routine will be the first step to you achieve your goals as quickly as possible!
```

Comment:
I think we could change this paragraph to this:

`
In this section you must inform your schedule, the time you will dedicate for your journey as a developer, after all, the studies routine will be your first step in order to achieve your goals as quickly as possible!
`